### PR TITLE
updating GTM tag

### PR DESCRIPTION
**Pull request by jdamiba**

### CODE REVIEW

File: _layouts/new_layout.html [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
+
   </head>
   {% include new_header.html %}
   {% include hero_sign_up.html %}
   {% include navigation_bar.html %}
   {% include make_chart_button.html %}
   <body>
-  <div class="container smallcontainer">
-    
+      <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-N6T2RXG"
```
Comment (nicolaskruchten) [inline _layouts/new_layout.html]: is this the only `<body>` tag?

File: _includes/_new/_page-components/_seo.html [hunk 2]
```diff
@@ -25,11 +25,8 @@
...
 <!-- Global site tag (gtag.js) - AdWords: 1009791370 -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=AW-1009791370"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'AW-1009791370');
-</script>
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```
Comment (nicolaskruchten) [inline _includes/_new/_page-components/_seo.html]: any other `<head>` tags?

File: _includes/head.html [hunk 3]
```diff
@@ -1,6 +1,21 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <!-- Global site tag (gtag.js) - AdWords: 1009791370 -->
+    <script async src=""https://www.googletagmanager.com/gtag/js?id=AW-1009791370""></script>
```
Comment (nicolaskruchten) [inline _includes/head.html]: similar to my comments in https://github.com/plotly/documentation/pull/1152 we'll need to ensure that we're not loading this tag here, and in any other place that calls this include.

File: 404.html [hunk 4]
```diff
@@ -5,39 +5,43 @@
 <!DOCTYPE html>
 <html>
     <head>
+				{% include google-tag-body.html %}
```
Comment (nicolaskruchten) [inline 404.html]: I assume this should be "head" not "body"

File: _includes/seo.html [hunk 5]
```diff
@@ -24,12 +24,5 @@
 <!-- Favicon -->
     <link rel="shortcut icon" href="/images/plotly-ico.png?v=2" />
 
-<!-- Global site tag (gtag.js) - AdWords: 1009791370 -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=AW-1009791370"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+    {% include google-tag-head.html %}
```
Comment (nicolaskruchten) [inline _includes/seo.html]: if you add this here, you have to make sure that you don't add it anywhere that also has `{% include seo.html %}`... for example in `_includes/header.html`

**Comment by jdamiba** [comment]
I removed the GTM include from `_includes/seo.html`, since the former is already present in its own `include` in every page where the latter is also included.

File: _layouts/two_column_layout.html [hunk 6]
```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
+      {% include google-tag-head.html %}
+      <script async="" src="/browser-sync/browser-sync-client.2.11.1.js"></script>
```
Comment (nicolaskruchten) [inline _layouts/two_column_layout.html]: is it normal that this browser-sync line snuck in here?

File: _layouts/old.html [hunk 7]
```diff
@@ -1,240 +0,0 @@
-<!DOCTYPE html>
```
Comment (nicolaskruchten) [inline _layouts/old.html]: we're sure this isn't being used anywhere?

File: _layouts/two_column_layout.html (hunk 6 above)
Comment (jdamiba) [reply _layouts/two_column_layout.html]: I did that on purpose. The `<head>`'s of all the other layout components include it, so it seemed like it made sense to also include it here.

File: _layouts/old.html (hunk 7 above)
Comment (jdamiba) [reply _layouts/old.html]: Yes, none of the posts have `layout: old`.

File: _layouts/two_column_layout.html (hunk 6 above)
Comment (nicolaskruchten) [reply _layouts/two_column_layout.html]: ok

**Comment by nicolaskruchten** [comment]
OK, I think this guy's good too 💃

**Pull request merged**